charts/refractr/templates/ingress-hook.yaml

```diff
@@ -71,36 +71,6 @@ subjects:
     name: refractr
 ---
 
-apiVersion: helm.fluxcd.io/v1
```

Comment:
Do you know why there is a HelmRelease inside a Helm Chart? Is this needed or is a bug?

If we need this, it has to be added to the Chart.yaml file as a dependency

Reply:
This was solved moving it to a dependency.